2019-04-25-coopdevs-chat.md

```diff
@@ -0,0 +1,132 @@
+    - CoopDevs
+- Open Questions (from [question bank](https://docs.google.com/spreadsheets/d/1vjnT0afFh-Ss1BCYZg7nuLyjwZqzRLsSzkhej_OdoFo/edit#gid=348272266))
+
+**Legend**  
+:star: Action  
+... Missed
+
+## Notes
+
+- Intros
+    - Ben: started working together with a group of folks on mesh+events in Toronto.
+    - dc: Research+organizing -- food and data justice, decentralization.
+    - patcon: developer, community organizing -- civic tech, adjacent to gov't
+    - pau: comp eng. ... bridging tech & social movements. hitech+coop. in barcelona region. katuma platform for regional supply chains. saas platform, then coop formation? ...
+    - ferran: ansible [& devops?]. accounting for coops → odoo (Open ERP)? coop telecom (somconnexio.coop).
+
+![@ben's sketch of coopdevs org structure](https://i.imgur.com/qgJNkme.png)
+
+- ben: recap of offline Barcelona pre-convo:
+    - CoopDev (CD), based in BCN, structure: 4owner+2workers, 1 year worker becomes member (probation)
+        - ferran+antonio = workers
+    - CD part of larger assoc., previously existed, 
+    - legal form of the wider org: "association". has president, money person, secretary and thats it.
+    - no right to economical activity. You can issue invoices but not much more.
+    - ppl can come close via the association (less legal structure, more social connection)
+    - to join professionally, have to join the co-op 
+    - larger assoc meets every tues for meetups? very loose
+    - totally separate legal entities. informal agreement between assoc members and co-op. but same project and single brand.
+        - one logo → http://coopdevs.org/
+        - same communication channels
+    - revenue
+        - assoc shouldn't need much money to run. to be associate it's just 10€. But coop feeds or helps it. stickers, events, etc.
+        - coop has customers and services and some grants in order to pay salaries (non-profit).
+- Most people are full-time or near
+- 60% billable work, 40% internal/donated/volunteer work
+    - "like a donation of cooperative to project"
+    - non-paid work, non-client work: work on community, communication "re-productive" :)
+        - "trying to get rid of this work"? Couldn't hear well. [name=patcon]
+    - timeoverflow example: 20h a month as a donation. One city council is paying for some feature but we out-sourced it from the coop.
+- Grant as a means to start and consolidate projects.
+    - ERP (resource planning) example: goal is that next year should be in market and switch to customer revenue v. grant
+    - lucky: city gov and catalan govt eager to fund coop and solidarity econ
+        - [research group at Catalunya university](https://www.uoc.edu/portal/en/in3/recerca/grups/digital_commons) are big fans of platform coop and them (pilot proj & EU funding)
+        - for Katuma digital platform coop
+        - coopdevs initated it as a kind of little incubator, each member has many projects
+- org has accumulated many projects
```

Comment:
Got permission to share this :)

```suggestion

![coopdevs projects](https://i.imgur.com/dozQl5n.jpg)

- org has accumulated many projects (above diagram shows members responsible for them)
```

Reply:
Niiiiice. Thanks Ben! Curious: know what green dots and exclamation marks mean?

Reply:
Also, feels maybe worth pointing back to the fact that ferran+antonio (who have much less lines of responsibility) are non-member workers, rather than member-owners that others are.